test/Analyzer.Tests/TSQLLintConfigurationTests.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using Codacy.TSQLLint.Configuration;
+using Xunit;
+using Newtonsoft.Json;
+
+namespace Codacy.TSQLLint.Tests
+{
+    public class TSQLLintConfigurationTests
+    {
+        [Fact]
+        public void JsonTest()

```

Comment:
please add tests for the failure scenarios

Reply:
What do you mean by that? This test tests if the configuration file is well parsed, not if its valid. So, this can actually take any values inside `TSQLLintConfiguration`. The validation of the configuration file is made by the tool side.